app/scripts/views/WorkspaceView.js

```diff
@@ -534,13 +534,14 @@ define(['backbone', 'Workspace', 'ConnectionView', 'MarqueeView', 'NodeViewTypes
           }
 
           nodeView = new NodeView({ model: nodeModel, workspaceView: this_, workspace: this_.model });
+          this_.watchNodeViewEvents(nodeView);
           this_.nodeViews[ nodeView.model.get('_id') ] = nodeView;
 
         }
 
         this_.$workspace.prepend( nodeView.$el );
+        this_.makeDraggable(nodeView);
         nodeView.render();
-        nodeView.makeDraggable();
```

Comment:
`makeDraggable` was called twice (in node's `initialize` as well) so it was redundant. but now this method is moved here and stops being called in `initialize`. Also it should be called before rendering